pfsimulator/parflow_lib/solver_richards.c

```diff
@@ -611,6 +612,26 @@ SetupRichards(PFModule * this_module)
                            sizeof(slope_filenames) / sizeof(slope_filenames[0]),
                            slope_filenames);
   }
+  
+  if (public_xtra->print_channelwidth)
+  {
+    strcpy(file_postfix, "wc_x");
+    WritePFBinary(file_prefix, file_postfix,
+                  ProblemDataChannelWidthX(problem_data));
+
+    strcpy(file_postfix, "wc_y");
+    WritePFBinary(file_prefix, file_postfix,
+                  ProblemDataChannelWidthY(problem_data));
+
+    static const char* slope_filenames[] = {
```

Comment:
Nitpick but these are not "slope_" filenames.   I'm fine with just using "filenames" here to reduce the "copy/paste forget to change name" errors.   A local "filenames" array should be reasonably clear.